pyrit/prompt_converter/azure_speech_text_to_audio_converter.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import logging
+import pathlib
+import os
+import uuid
+
+import azure.cognitiveservices.speech as speechsdk
+from pyrit.memory.memory_models import PromptDataType
+from pyrit.prompt_converter import PromptConverter
+from pyrit.common import default_values
+from pyrit.common.path import RESULTS_PATH
+
+logger = logging.getLogger(__name__)
+
+
+class AzureSpeechTextToAudioConverter(PromptConverter):
+    """
+    The TextToAudio takes a prompt and generates a
+    wave file.
```

Comment:
```suggestion
    The TextToAudio takes a prompt and generates a wave file.
```
The Google docstring format starts with a single line description.